test_files/web_animation.untyped/web_animations.d.ts

```diff
@@ -0,0 +1,72 @@
+// Type definitions for web-animations-js 2.2
```

Comment:
Does this test have to have all web-animations? Can you prune it to a more minimal part that is tested against closure_externs.js

Reply:
Also this needs a comment on what is it testing.